packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -187,7 +187,7 @@ fi
 {{- end }}
       --light.serve={{- if kindIs "invalid" .light_serve -}}90{{- else -}}{{- .light_serve -}}{{- end }} \
       --light.maxpeers={{- if kindIs "invalid" .light_maxpeers -}}1000{{- else -}}{{- .light_maxpeers -}}{{- end }} \
-      --maxpeers {{ .maxpeers | default 1200 }} \
+      --maxpeers={{- if kindIs "invalid" .maxpeers -}}1200{{- else -}}{{- .maxpeers -}}{{- end }} \
```

Comment:
This may be a question for another PR, but I don't like the idea of the `common` template defaulting to a full node, and with so many peers as well.  I'm not sure if or when the default is used and whether we rely on it.  Certainly we don't want to unwitting change existing nodes which depend on it.  But in general I think a more reasonable approach (especially once we upgrade to v1.3.0, where light.serve default is 0) would be to default to not specifying these flags and relying on the geth defaults.

Reply:
Seems good to me. I'd rather open a new PR for this and discuss there if there could be any impact for other usages of this snippet (like oracles, komenci or other full node usages).